hack/verify-flags/verify-uniq.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+function check() {
+  d=$(sort $1 | uniq | diff $1 - || true)
+  if [[ -n "${d}" ]]; then
+    echo "Run \"sort $1 | uniq | tee $1 > /dev/null\"" >&2
```

Comment:
That'll wipe the file! I guess you have to go through tee if you want to pipe back into a file you're reading from.